client/webpack.config.dev.js

```diff
@@ -58,21 +60,24 @@ module.exports = {
       {
         test: /\.otf$/, loader: 'file-loader',
       },
+      {
+        test: /\.otf$/, loader: 'file-loader',
+      },
       {
         test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: 'url?limit=10000&mimetype=image/svg+xml',
       },
     ],
   },
   devServer: {
     host: '0.0.0.0',
-    publicPath: 'http://0.0.0.0:9000/',
+    publicPath: 'http://0.0.0.0:3000/',
```

Comment:
This is going to be a breaking change and everyone should be aware of this.

If possible, let this be reflected in the README.

Reply:
Was supposed to change that before I pushed, my port 9000 is being used by another application.